dto.c

```diff
@@ -43,6 +45,8 @@
  * a hang (e.g., memset --> malloc --> alloc library calls memset --> memset)
  */
 #define MAX_WQS 16
+#define MAX_NUMA_NODES 16
+#define MAX_WQS_IN_DSA_DEVICE 8
```

Comment:
SPR has 8 WQs per DSA but future platforms will have more DSAs. GNR has 16 WQ per DSA. can we use MAX_WQS that is already defined?